MozillaFoundation/foundation.mozilla.org
# Localized attribute removed from mozfest_footer.html

# Description

This PR removes `.localized` attribute from `mozfest_footer.html` to avoid duplicity in querying for localized content. The logic was already implemented on [_get_signup](https://github.com/MozillaFoundation/foundation.mozilla.org/blob/a284b3176c223bf8589076e061a0400f73286b4a/network-api/networkapi/mozfest/models.py#L224).

Link to sample test page:
Related PRs/issues: https://github.com/MozillaFoundation/foundation.mozilla.org/issues/9882

# How to test
- Pull branch locally.
- Go to mozfest.localhost footer, and change the locale.
- Make sure the content is translated properly.

![image](https://github.com/user-attachments/assets/51dd4fa6-e1f8-432b-85d6-2217596825e4)
![image](https://github.com/user-attachments/assets/0516735c-429c-4591-8b4b-de75d94e94b7)

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-1523)
